node-lib/src/options.rs

```diff
@@ -80,6 +89,11 @@ pub struct RunOptions {
     #[clap(long, short, action = clap::ArgAction::SetTrue)]
     pub clean_data: Option<bool>,
 
+    /// Log to a file
+    #[clap(long, action = clap::ArgAction::Set)]
+    #[arg(hide = true)]
+    pub log_to_file: Option<bool>,
```

Comment:
Should we hide it?

(If not, the description should be more detailed, mentioning that the default value depends on the application and where the logs will be located).

Reply:
but it's already hidden

Reply:
> but it's already hidden

Yeah, but I'm wondering whether it should be, because it's potentially useful for node daemon too

Reply:
ok maybe you're right, removed hidden